content/english/python-tensorflow/answer_key.md

```diff
@@ -217,10 +207,8 @@ array([1.1623413e-08, 2.3745208e-08, 3.5589391e-08, 1.2773025e-09,
 ![](https://i.imgur.com/td1b4Jk.png)
```

Comment:
This is an image, and ideally all images should have alt text that describes the image adequately, for example

```suggestion
![Showing the highest and lowest values of the probability array](https://i.imgur.com/td1b4Jk.png)
```

If you have time, please take a look at adding alt text to all the images in this workshop.